### PR TITLE
Kk/withdraw gas price

### PR DESCRIPTION
**Pull request by krzkaczor**

### CODE REVIEW

File: app/components/modals/txSender/shared/TxPending.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  txHash: string;
+}
+
+export const TxPending: React.SFC<ITxPendingProps> = ({ blockId, txHash }) => (
+  <div className="text-center">
+    <SpinningEthereum className="mb-3" />
+
+    <h3 className={styles.title}>Pending transaction</h3>
+
+    <p>
```
Comment (Moejoe90) [inline app/components/modals/txSender/shared/TxPending.tsx]: Can we add this to translations?

File: webpack/proxy-urls.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ function generateProxyConfig(backendUrl) {
   const base = {
     "/node": {
       targetLocal: "http://localhost:8545",
-      targetRemote: "http://localhost:8545",
+      targetRemote: "https://platform.neufund.io/nodes/private",
```
Comment (Moejoe90) [inline webpack/proxy-urls.js]: Does this mean there is no need for ganache with this change?

File: app/components/modals/txSender/withdraw-flow/Withdraw.tsx [hunk 3]
```diff
@@ -57,3 +78,21 @@ export const Withdraw: React.SFC<IWithdrawProps> = ({ onAccept }) => (
     </Formik>
   </div>
 );
+
+export const GasComponent: React.SFC<IGasState> = ({ gasPrice, error }) => {
+  if (error) {
+    return <WarningAlert>Error occured while getting optimal GAS price.</WarningAlert>;
```
Comment (Moejoe90) [inline app/components/modals/txSender/withdraw-flow/Withdraw.tsx]: Also this can you add it to translatiosn

File: webpack/proxy-urls.js (hunk 2 above)
Comment (krzkaczor) [reply webpack/proxy-urls.js]: ah fuck maybe i should not commit this. Let's talk about it during daily

**Pull request merged**